utils/src/config.rs

```diff
@@ -398,9 +398,7 @@ pub mod tls {
                 let cert_option = cert_result.ok();
                 Ok(TlsConfig::prefer(cert_option))
             }
-            TlsConnectMode::Require if cert_ok => {
-                // Unwrapping the result since we've verifed the Result is Ok in
-                // the guard
+            TlsConnectMode::Require => {
```

Comment:
Just so I understand: this means if the mode is require, a cert is needed, right? And if the mode is none of the specified cases, we allow communication to postgres without a cert?

Reply:
So, the idea is that `allow`, `prefer`, and `require` *optionally* allow for a `certificate` option to be supplied.

See my comment here https://github.com/joyent/rust-cueball/pull/51#discussion_r433370692

Reply:
Perfect, thanks!